OpenRA.Mods.Common/Traits/GivesExperience.cs

```diff
@@ -41,14 +41,16 @@ public void Killed(Actor self, AttackInfo e)
 
 			var valued = self.Info.TraitInfoOrDefault<ValuedInfo>();
 
-			// Default experience is 100 times our value
 			var exp = info.Experience >= 0
 				? info.Experience
-				: valued != null ? valued.Cost * 100 : 0;
```

Comment:
You can (and should!) test this by removing your manual yaml changes and running the updater over the default mods.